spec/git_reflow/all-approval/git_reflow_spec.rb

```diff
@@ -18,6 +18,11 @@
   let(:existing_pull_request)  { GitReflow::GitServer::GitHub::PullRequest.new existing_pull_requests.first }
 
   before do
+
```

Comment:
I see.  We can keep the separation, but I still see value in keeping the original spec and remove any scenarios that aren't affected by the configuration change (e.g. the `stage` command).  More tests _can_ hurt since we have to maintain them.